src/profile/MIANotification/MIANotification.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { FC } from 'react';
+
+import i18n from '../../_translations/i18n';
+import { formatDate } from '../../_utils/dateHelpers';
+import { TCardSystemMembership } from '../_models';
+import * as Styled from './style';
+
+type TProps = {
+  socialTariffInfo: TCardSystemMembership['socialTariff'];
+};
+
+const MIANotification: FC<TProps> = ({ socialTariffInfo }) => {
+  return (
+    <Styled.NotificationContainer>
+      <Styled.CloseButton name="KansenTarief" size={25} />
+      <Styled.MIAText color="darkGreen">{i18n.t('PROFILE.MIA')}</Styled.MIAText>
+      <Styled.ExpirationText color="darkGreen" size="small">{`${i18n.t('PROFILE.VALID_UNTIL')} ${formatDate(
+        new Date(socialTariffInfo?.endDate),
```

Comment:
The optional chaining operator indicates that `socialTariffInfo` is possibly undefined. But it's a required prop. 🤔 

Is there a use case where `socialTariffInfo?.endDate` is emtpy? In that case the `new Date(undefined)` will display 'Invalid Date'. Might need some error handling..